keyboards/annepro2/c15/rules.mk

```diff
@@ -39,3 +39,7 @@ RAW_ENABLE = yes
 MIDI_ENABLE = no
 VIRTSER_ENABLE = no
 COMBO_ENABLE = no
+
+# Custom led mcu wakeup delay
+ANNEPRO2_LED_MCU_WAKEUP_DELAY = 15
```

Comment:
I'd increase it to 100ms initially. It's not really noticeable. The boards starts slowly anyway and there's at least one person that had it working with 100 (maybe 50 would be enough though). 
You could put this into config.h, opt_defs are not required then. Not sure if it's better.

Reply:
(More pro approach would be to send it periodically until shine sends us a status...)